Image "linux-armv7a" build failure

See https://app.circleci.com/pipelines/github/thewtex/dockcross/11/workflows/61200910-b08a-4b68-b32c-d011d7241a54/jobs/1822/steps

```
[EXTRA]    Retrieving 'expat-2.2.0'
[ERROR]   
[ERROR]  >>
[ERROR]  >>  Build failed in step 'Retrieving needed toolchain components' tarballs'
[ERROR]  >>        called in step '(top-level)'
[ERROR]  >>
[ERROR]  >>  Error happened in: do_expat_get[scripts/build/companion_libs/210-expat.sh@12]
[ERROR]  >>        called from: do_companion_libs_get[scripts/build/companion_libs.sh@15]
[ERROR]  >>        called from: main[scripts/crosstool-NG.sh@604]
[ERROR]  >>
[ERROR]  >>  For more info on this error, look at the file: 'build.log'
[ERROR]  >>  There is a list of known issues, some with workarounds, in:
[ERROR]  >>      '/dockcross/crosstool/ct-ng/prefix/share/doc/crosstool-ng//B - Known issues.txt'
[ERROR]  >>
[ERROR]  >>  If you feel this is a bug in crosstool-NG, report it at:
[ERROR]  >>      https://github.com/crosstool-ng/crosstool-ng/issues/
[ERROR]  >>
[ERROR]  >>  Make sure your report includes all the information pertinent to this issue.
[ERROR]  >>  Read the bug reporting guidelines here:
[ERROR]  >>      http://crosstool-ng.github.io/support/
[ERROR]   
[ERROR]  (elapsed: 0:33.71)
[00:34] / /dockcross/crosstool/ct-ng/prefix/bin/ct-ng:147: recipe for target 'build' failed
make: *** [build] Error 1
The command '/bin/sh -c mkdir /dockcross/crosstool && cd /dockcross/crosstool && /dockcross/install-crosstool-ng-toolchain.sh   -p "${XCC_PREFIX}"   -c /dockcross/crosstool-ng.config && rm -rf /dockcross/crosstool /dockcross/install-crosstool-ng-toolchain.sh' returned a non-zero code: 2
make: *** [linux-armv7a] Error 2

```

--- a/common-manylinux.crosstool
+++ b/common-manylinux.crosstool
@@ -47,6 +47,7 @@ ENV XCC_PREFIX=/usr/xcc
 # for users.
 COPY \
   imagefiles/install-crosstool-ng-toolchain.sh \
+  imagefiles/crosstool-ng-expat.patch \
   manylinux2014-aarch64/crosstool-ng.config \
   /dockcross/
 

--- a/common.crosstool
+++ b/common.crosstool
@@ -32,6 +32,7 @@ ENV XCC_PREFIX=/usr/xcc
 # for users.
 COPY \
   imagefiles/install-crosstool-ng-toolchain.sh \
+  imagefiles/crosstool-ng-expat.patch \
   crosstool-ng.config \
   /dockcross/
 

--- a/imagefiles/crosstool-ng-expat.patch
+++ b/imagefiles/crosstool-ng-expat.patch
@@ -1,0 +1,11 @@
+--- crosstool-ng-crosstool-ng-1.23.0/scripts/build/companion_libs/210-expat.sh	2021-04-05 13:55:31.047130000 +0000
++++ crosstool-ng-crosstool-ng-1.23.0/scripts/build/companion_libs/210-expat.sh.new	2021-04-05 13:57:13.841170000 +0000
+@@ -10,7 +10,7 @@
+ 
+ do_expat_get() {
+     CT_GetFile "expat-${CT_EXPAT_VERSION}" .tar.gz    \
+-               http://downloads.sourceforge.net/project/expat/expat/${CT_EXPAT_VERSION}
++               https://github.com/libexpat/libexpat/releases/download/R_${CT_EXPAT_VERSION//./_}
+ }
+ 
+ do_expat_extract() {

--- a/imagefiles/install-crosstool-ng-toolchain.sh
+++ b/imagefiles/install-crosstool-ng-toolchain.sh
@@ -61,6 +61,7 @@ REV=1.23.0
 curl -# -LO \
   "https://github.com/crosstool-ng/crosstool-ng/archive/crosstool-ng-${REV}.tar.gz"
 tar -xf "crosstool-ng-${REV}.tar.gz"
+patch crosstool-ng-crosstool-ng-1.23.0/scripts/build/companion_libs/210-expat.sh -i /dockcross/crosstool-ng-expat.patch
 cd "crosstool-ng-crosstool-ng-${REV}"
 
 # Bootstrap and install the tool.

